src/controllers/trainee/validation.ts

```diff
@@ -0,0 +1,56 @@
+const config = {
+    get: {
+        skip: {
+            required: false,
+            default: 0,
+            number: true,
+            in: ['query'],
+            errorMessage: 'Skip is invalid'
```

Comment:
Is this is a validation file or config file. Use proper naming convention

Reply:
it is a validation file and inside a constant config is defined as per the guidelines of user story on redmine